src/Keycloak.tsx

```diff
@@ -0,0 +1,108 @@
+import Keycloak from 'keycloak-js';
+import React from 'react';
+
+import { Auth, AuthContext } from '@rhoas/app-services-ui-shared';
+
+export let keycloak: Keycloak.KeycloakInstance | undefined;
+
+/**
+ * Get keycloak instance
+ *
+ * @return an initiated keycloak instance or `undefined`
+ * if keycloak isn't configured
+ *
+ */
+export const setKeycloakInstance = async () => {
+  if (!keycloak) await init();
+};
+
+/**
+ * Initiate keycloak instance.
+ *
+ * Set keycloak to undefined if
+ * keycloak isn't configured
+ *
+ */
+export const init = async () => {
+  try {
+    /* Keycloak configuration using OB authentication.
+     * Thiese params will change after sso is implemented.
+     * See https://issues.redhat.com/browse/MGDOBR-466
+     */
+    keycloak = new (Keycloak as any)({
+      realm: 'event-bridge-fm',
+      url: 'https://keycloak-event-bridge-prod.apps.openbridge-dev.fdvn.p1.openshiftapps.com/auth/',
+      clientId: 'event-bridge',
+    });
+    if (keycloak) {
+      await keycloak.init({
+        onLoad: 'login-required',
+      });
+    }
+  } catch (e) {
+    keycloak = undefined;
+    console.warn(
+      'Auth: Unable to initialize keycloak. Client side will not be configured to use authentication',
+      e
+    );
+  }
+};
+
+/**
+ * Use keycloak update token function to retrieve
+ * keycloak token
+ *
+ * @return keycloak token or empty string if keycloak
+ * isn't configured
+ *
+ */
+export const getKeyCloakToken = async (): Promise<string> => {
+  await keycloak?.updateToken(50);
```

Comment:
what is `50`?

Reply:
It's the token minimum validity in seconds. If the token is going to expire within 50s it will be refreshed.

Reply:
I expect to have a constant with a meaningful name for it or at least a comment which explains why 50 it's the correct value and not 60, 70, 80, 90, 100...

Reply:
Fair point :+1:  

I moved the value to a constant and added a description there.

There's no "correct" value. The only important thing is it should be lower than the token lifespan. For our keycloak instance the lifespan is very high, 10 hours. It will be 15 minutes with sso.

To clarify, this keycloak setup is only used for local development of the standalone SE app.
When the SE module is imported within app services, it will retrieve tokens from there using the auth hook.